include/pagmo/algorithm.hpp

```diff
@@ -69,7 +69,7 @@ namespace pagmo
 /**
  * This type trait will be \p true if \p T provides a method with
  * the following signature:
- * @code
+ * @code{.unparsed}
```

Comment:
doxygen was being rather inconsistent about the automatic links it would generate in the code section (e.g., it would link to pagmo::vector_double correctly but it would fail to generate a link for, e.g., std::vector<vector_double>). Plus my experience with sphinx and breathe is that they mess up badly the syntax highlighting generate from doxygen code blocks.

So the idea is just to avoid having doxygen/sphinx try to parse anything and just give us a dumb code block.